digitalearthau/submit/ingest.py

```diff
@@ -58,12 +59,17 @@ def do_qsub(product_name, year, queue, project, nodes, walltime, name):
     cmd = test % dict(taskfile=taskfile)
     if click.confirm('\n' + cmd + '\nRUN?', default=False):
         subprocess.check_call(cmd, shell=True)
+    
+    product_changes_flag = ''
```

Comment:
Could also be written as:
`product_changes_flag = '--allow-product-changes' if allow_product_changes else ''`